spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/sampler/SamplerAutoConfiguration.java

```diff
@@ -40,39 +43,48 @@
 
 	@Bean
 	@ConditionalOnMissingBean
-	Sampler sleuthTraceSampler() {
+	Sampler sampler() {
```

Comment:
We've been meeting descriptive on names so there isn't collisions with user bean names

Reply:
sure thing. I'll put a comment. We do have places where we intentionally use fixed names, and there's no use case for multiple beans. Do you know why it would matter in this case when it is ConditionalOnMissingBean?

Reply:
no worries if you don't. the comment is probably good enough. Thanks!

Reply:
I undid the comment as the bean names weren't consistent here either. wild guess is we either meant to be consistent, but accidentally weren't, or we meant not to be consistent in order to figure out possibly which autoconfiguration branch led to that bean.